src/module-elasticsuite-core/Model/System/Message/WarningAboutClusterGhostIndices.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * DISCLAIMER
+ * Do not edit or add to this file if you wish to upgrade Smile Elastic Suite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCore
+ * @author    Romain Ruaud <romain.ruaud@smile.fr>
```

Comment:
You can put your name :)

Reply:
I always forget to unwrap all collapsed annotations ;-)